pkgs/net-doc/net/scribblings/ftp.scrbl

```diff
@@ -127,10 +136,15 @@ is intended to limit polling.
                                       #f])
          void?]{
 
-Upload @racket[file-path] to the server's current directory using the same name. 
-If the file already exists in the local directory, it is replaced.
-The @racket[progress-proc] argument is used in the same way as in @racket[ftp-download-file],
-but to report uploaded bytes instead of downloaded bytes.}
+When @racket[port] is @racket[#f], upload @racket[file-path] to the server's
+current directory using the same name.  If the file already exists in the local
```

Comment:
It looks like the original documentation had a mistake. This line should say "remote" instead of "local", right?

Reply:
I didn't spot it either. Definitely it should say "remote".